src/Makefile.am

```diff
@@ -7,3 +7,7 @@ if ENABLE_QT
 endif
 
 SUBDIRS=libprojectM NativePresets ${PROJECTM_SDL_SUBDIR} ${PROJECTM_QT_SUBDIR}
+
+# system headers/libraries/data to install
+# for compatibility reasons here as nobase_include
+nobase_include_HEADERS = libprojectM/projectM.hpp libprojectM/Common.hpp libprojectM/dlldefs.h libprojectM/event.h libprojectM/fatal.h libprojectM/PCM.hpp
```

Comment:
These were added here - https://github.com/projectM-visualizer/projectm/pull/53#discussion-diff-180316275R15
What's the difference? What are the compatibility issues?

Reply:
With CMake the files were installed to:

/usr/include/libprojectM

With Autotools all files are installed to

/usr/include

This is what this patch changes to avoid breakages in projects depending on projectM. Unfortunately it  breaks make dist (haven't tested - my bad). I will try to fix and come back when I have something better. As said that might last because it is a just for fun issue to me and during the week there is not much time left for pure fun stuff...

How about a resend with pkg-config patch only and see if that passes Travis. If OK you can take that one before.

Reply:
the files should definitely all be installed to /usr/include/libprojectM, I'm pretty sure. So if they are now, great! Just update the test and we should be good to go. And do test out `make dist` if you get a chance.